examples/resources/trocco_label/resource.tf

```
@@ -0,0 +1,5 @@
+resource "trocco_label" "example" {
+  name        = "label name"
+  description = "label description"
+  color       = "#FFFFFFF"
```

When I tried to `terraform plan` with this example directly, I got a validation error.
> │ Error: Invalid Attribute Value Match
│ 
│   with trocco_label.example,
│   on example.tf line 72, in resource "trocco_label" "example":
│   72:   color       = "#FFFFFFF"
│ 
│ Attribute color must be in format #RRGGBB or #RGB, got: #FFFFFFF

```suggestion
  color       = "#FFFFFF"
```

---

fix applied and doc regenerated 🙇